sympy/matrices/matrices.py

```diff
@@ -299,23 +299,66 @@ def _setitem(self, key, value):
             return
 
     def copy(self):
+        """
+        Return the copy of a matrix expression.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> A = Matrix(2, 2, [1, 2, 3, 4])
+        >>> A.copy()
+        Matrix([
+        [1, 2],
+        [3, 4]])
+
+        """
         return self._new(self.rows, self.cols, self._mat)
 
     def trace(self):
+        """
+        Return the trace of a matrix expression ie. the sum of the
+        elements on the diagonal.
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> A = Matrix(2, 2, [1, 2, 3, 4])
+        >>> A.trace()
+        5
+
+        """
         if not self.is_square:
             raise NonSquareMatrixError()
         return self._eval_trace()
 
     def inv(self, method=None, **kwargs):
+        """
+        Return the inverse of a matrix using Gaussian elimination by default.
+
+        Also, by setting method='LU' we can specify it to be done using LU decomposition.
+
+        However, if determinant of matrix is zero, a ValueError is raised as the inverse for that
```

Comment:
updated ie. -> i.e.
updated ValueError --> ``ValueError``
updated Return --> Returns
updated on the diagonals -> diagonal elements